pkg/ext-proc/main.go

```diff
@@ -156,3 +191,25 @@ func main() {
 	s.Serve(lis)
 
 }
+
+var onlyOneSignalHandler = make(chan struct{})
+
+// SetupSignalHandler registered for SIGTERM and SIGINT. A context is returned
+// which is cancelled on one of these signals. If a second signal is caught,
+// the program is terminated with exit code 1.
+func SetupSignalHandler() context.Context {
+	close(onlyOneSignalHandler) // panics when called twice
+
+	c := make(chan os.Signal, 2)
+	ctx, cancel := context.WithCancel(context.Background())
+	shutdownSignals := []os.Signal{os.Interrupt, syscall.SIGTERM}
```

Comment:
is os.Interrupt the same as syscall.SIGINT which we used above? if so, perhaps use the same variable in both places